packages/cli/src/formatters/utils/getScoring.ts

```diff
@@ -0,0 +1,68 @@
+import { SEVERITY_MAP } from '@stoplight/spectral-core';
+import { DiagnosticSeverity } from '@stoplight/types';
+import { ScoringConfig, ScoringTable, ScoringSubtract } from '../types';
+import * as path from '@stoplight/path';
+import fs from 'fs';
+
+export const getScoringConfig = (scoringFile?: string): ScoringConfig | undefined => {
+  if (scoringFile === void 0) {
+    return undefined;
+  } else if (!path.isAbsolute(scoringFile)) {
+    scoringFile = path.join(process.cwd(), scoringFile);
+  }
+
+  const scoringConfig: ScoringConfig = JSON.parse(fs.readFileSync(scoringFile, 'utf-8')) as ScoringConfig;
```

Comment:
```suggestion
  const scoringConfig: ScoringConfig = JSON.parse(fs.promises.readFile(scoringFile, 'utf8'));
```

I'd also say we should impose some validation.
We could define a JSON Schema for the format and use Ajv for that purpose.
What do you think?

Reply:
We've got your suggestion
For the validation ... We guess that could be a duplicated validation, with type defined on _cli/src/formatters/types.ts_ should be enough to validate input file, but, if anybody thinks like you about this, we could add this